src/foam/u2/md/ActionView.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+foam.CLASS({
+  package: 'foam.u2.md',
+  name: 'ActionView',
+  extends: 'foam.u2.View',
+
+  properties: [
+    'action',
+    [ 'nodeName', 'paper-button' ],
+    {
+      class: 'Boolean',
+      name: 'raised',
+      documentation: 'Controls the "raised" attribute in Polymer. Defaults ' +
+          'to false; set this to true for a raised, bordered button.'
+    },
+    {
+      name: 'label',
+      expression: function(action) { return action.label; }
+    }
+  ],
+
+  methods: [
+    function initE() {
+      this.cssClass(this.myCls())
+          .attrs({
+            disabled: ! this.action.isEnabled ? false :
+                this.action.createIsEnabled$(this.data$).map(function(e) {
+                  return e ? false : 'disabled';
+                })
+          })
+          .on('click', this.click)
+          .add(this.label$);
+
+      if ( this.action.isAvailable ) {
```

Comment:
See the recent changes to the regular ActionView. unavailable is now the CSS class we use to toggle and if isAvailable isn't set then you don't need to do anything.